.github/workflows/checkstyle.yaml

```diff
@@ -34,11 +34,11 @@ jobs:
     - name: CheckABI
       id: CheckABI
       run: |
-        sudo docker run -v $(pwd):/source ghcr.io/openzfs/libabigail make checkabi
+        sudo docker run -v $(pwd):/source ghcr.io/xnox/libabigail make checkabi
     - name: StoreABI
       if: failure() && steps.CheckABI.outcome == 'failure'
       run: |
-        sudo docker run -v $(pwd):/source ghcr.io/openzfs/libabigail make storeabi
+        sudo docker run -v $(pwd):/source ghcr.io/xnox/libabigail make storeabi
     - name: Prepare artifacts
```

Comment:
Note above workflow changes are in a stand alone commit, that should be reverted once https://github.com/openzfs/libabigail-docker/pull/2 is merged and deployed.